grammatical and info fixes on roundstart tips + adds a single space to the valhalla mobspawn message (#13768)
tgstation/TerraGov-Marine-Corps: strings/tips/marine.txt

Anything can be a weapon if you put your mind to it.
You can check your wounds to see if they have been bandaged or salved by clicking on yourself while in help intent.
You can check what part of your body is bleeding by shift-clicking on your person.
Removing weeds as a marine means that xenomorphs cannot take advantage of the battlefield; your standard boot knife is great for doing this.
Blue weeds and purple sticky resin slow you down. Get rid of them!
During Self Destruct, green control rods are armed control rods. Each rods take 3 minutes to rise, forcing marines to hold Self Destruct for a total of 20 minutes. Collect all six!
A fit marine can carry many weapons: in their armor, belt, even hanging from their back.
Requisitions has many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's in store and you may be pleasantly surprised. DISCLAIMER: Extra gears' availability is dependent on supply.
Cargo points are valuable: some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.

> As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself onto them.

Reviewer: should probs mention jumping.

Author: i went ahead and made it a new tip, just because i think combining the two would be too long, and i wanted to flesh out more than just "you can also jump". i can also just delete the "specialclicking" tip since i think jumping is the superior way to move across objects

```You can jump over waist-high obstacles like platforms, tables, and windowframes, for a small stamina cost, or, if a xeno, for a two-second cooldown. Set your key to jump under the Game Preferences menu!```

let me know if you want the wording edited at all. i also forgot, but it has to be binded, there's no default key, correct?

Reviewer: but default it should be space or something.

Anything can be a weapon if you put your mind to it.
You can check your wounds to see if they have been bandaged or salved by clicking on yourself while in help intent.
You can check what part of your body is bleeding by shift-clicking on your person.
Removing weeds as a marine means that xenomorphs cannot take advantage of the battlefield; your standard boot knife is great for doing this.
Blue weeds and purple sticky resin slow you down. Get rid of them!
During Self Destruct, green control rods are armed control rods. Each rods take 3 minutes to rise, forcing marines to hold Self Destruct for a total of 20 minutes. Collect all six!
A fit marine can carry many weapons: in their armor, belt, even hanging from their back.
Requisitions has many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's in store and you may be pleasantly surprised. DISCLAIMER: Extra gears' availability is dependent on supply.

> Cargo points are valuable: some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.

Reviewer 2: I don't think we even have these in req anymore

As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL+click) on them or drag-clicking yourself onto them.
You can jump over waist-high obstacles like platforms, tables, and window frames, for a small stamina cost (Default: Spacebar).
The Terra Experimental laser rifles have a mode selector that can be switched using the Unique Action command (Default: Spacebar). Some modes are better than other modes in specific circumstances.
You can remove armor pieces (leg pieces, arm pieces and chest pieces) and armor modules (Valkyrie, Baldur, etc.) from the XM-02 Combat Exoskeleton by Alt+clicking the exoskeleton.
You can customize armor pieces and the exoskeleton helmet using any kind of facepaint for a personal touch. Paint the armor pieces before you put them in the exoskeleton!

> If an M40 FLDP grenade collides into any mob when thrown, the mob will be lit in a light fire. Be careful throwing flares.

Reviewer 2: ```suggestion
If an M40 FLDP grenade collides with any mob when thrown, the mob will be lit in a light fire. Be careful throwing flares.
```

SWAT masks ordered from Requisitions can block only one larval facehugger attack. If they successfully blocked an attempted attack, replace them immediately.

> Drag dead xenomorphs onto the Automated Storage and Retrieval System (ASRS) pad inside cargo. Once the pad lowers with the applicable bodies, you will gain requisition points.

Reviewer 2: ```suggestion
Drag dead xenomorphs onto the Automated Storage and Retrieval System (ASRS) pad inside requisitions. Once the pad lowers with the applicable bodies, you will gain requisition points.
```

Not being ready at the start of the round means xenos get more burrowed larva, and that you don't get to play as quickly.
As a squad marine, you can vend seperate Jaeger Pattern armor pieces and modules for the XM-02 Combat Exoskeleton from a Surplus Equipment Vendor. It is a modular piece of equipment after all.
As a squad marine, you can practice using your weapons on the Firing Range.
As a squad marine, you can handle every type of weapon featured in the game.
As a squad marine, when you are in a tight situation, do not be a hero! Whenever possible, recover and re-arm and leave the enemies for another try.
As a squad marine, do not be afraid to take risks. See a xeno that is almost dead and slowly retreating? Take the chance to charge and kill it! You have better chances to survive with a squadmate with you.
As a squad marine, you can help out your squad by bringing extra ammunition and food to share.
As a squad marine, you can take two two-handed guns, one in the suit slot and one on your back.

> As a squad marine, you can get attachments from the weapons vendors, usually near preparations.

Reviewer 2: ```suggestion
As a squad marine, you can get attachments from the weapons vendors, usually in preparations.
```

As a corpsman, hypervene is useful. Use it to purge deadly neurotoxins and larva-boosting growth hormones.
The B18 armor from Requisitions has an built-in medical system. The Valkyrie Automedical Armor System can do the same for other armors.
The V1 tarp (available from Requisitions) will blend you into the tile you're on once you cover yourself with it.
The FL-84 flamethrower has an inbuilt fire extinguisher as an attachment.
As the PO, pay attention to the radio! Who knows if someone needs immediate surgery or evacuation until their fate becomes worse?
As the Captain, you have a unique Mateba revolver that takes a chunk of a target's healthpool down. However, you are important for completing the mission, so do not participate in active combat using the Mateba!
Even if a patient dies a second after being successfully revived by a defibrillator, this resets the time it takes for them to become permanently brain dead.
Don't forget that xenomorphs spawn infinitely during the Crash gamemode. Don't waste too much time hunting xenomorphs down!
The AI and synthetics can talk together on their own channel using the :n key.

> You can memorialize fallen marines by using their dogtags on the ship's memorial! Memorialized solders will be displayed at the end of the round.

Reviewer: ?

Author: will fix the :n --> .n back to the :n.

unless you meant the memorial thing; that hasn't worked in months.

As a corpsman, remember to bring extra kelotane or dermaline to heal burn damage. Xenos love acid and burning marines.
As a corpsman, hypervene is useful. Use it to purge deadly neurotoxins and larva-boosting growth hormones.
The B18 armor from Requisitions has an built-in medical system. The Valkyrie Automedical Armor System can do the same for other armors.
The V1 tarp (available from Requisitions) will blend you into the tile you're on once you cover yourself with it.
The FL-84 flamethrower has an inbuilt fire extinguisher as an attachment.
As the PO, pay attention to the radio! Who knows if someone needs immediate surgery or evacuation until their fate becomes worse?
As the Captain, you have a unique Mateba revolver that takes a chunk of a target's healthpool down. However, you are important for completing the mission, so do not participate in active combat using the Mateba!
Even if a patient dies a second after being successfully revived by a defibrillator, this resets the time it takes for them to become permanently brain dead.
Don't forget that xenomorphs spawn infinitely during the Crash gamemode. Don't waste too much time hunting xenomorphs down!

> The AI and synthetics can talk together on their own channel using the :n prefix.

Reviewer 2: ```suggestion
The AI and synthetics can talk together on their own channel using the :n or .n prefix.
```

Sleeping will slowly heal cloneloss.
As a mech-pilot, speed is not very important.
As a mech-pilot, keep a close eye on your teamates. You lead the front!
As a mech-pilot, the only weapon that has IFF is the sniper, everything else can friendly-fire marines.
As a mech-pilot, make sure to repair and reload often.
As a flamer, you are a very valuable target. You clear resin walls and doors for marines to push. Just don't push marines into fire.
Back fuel-tanks can be refilled via welding kits.
White Phosphorus is very dangerous.
Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
Gigachads leave the autodoc on automatic.
You can fit tiny items into your helmet.
The time it takes for you to wield your gun can be the difference between life and death.
Switching to your sidearm is faster than reloading.
You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.

> With your sidearm in hand, click a pistol pouch to instantly reload.

Reviewer: it specifically needs to be right mouse button. Otherwise you're just storing it.

Sleeping will slowly heal cloneloss.
As a mech-pilot, speed is not very important.
As a mech-pilot, keep a close eye on your teamates. You lead the front!
As a mech-pilot, the only weapon that has IFF is the sniper, everything else can friendly-fire marines.
As a mech-pilot, make sure to repair and reload often.
As a flamer, you are a very valuable target. You clear resin walls and doors for marines to push. Just don't push marines into fire.
Back fuel-tanks can be refilled via welding kits.
White Phosphorus is very dangerous.
Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
Gigachads leave the autodoc on automatic.
You can fit tiny items into your helmet.
The time it takes for you to wield your gun can be the difference between life and death.
Switching to your sidearm is faster than reloading.
You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
With your sidearm in hand, right-click a pistol pouch to instantly reload.
You can turn the safety of your gun on.
Depending on the tactical situations, smoke grenades can do more harm than good.
A miner without a turret is a sitting duck.

> Claymores often one-shot T1s.

Reviewer 2: This one isn't even true.

The Baldur armor module significantly increases your light output. Become the flashlight you always wanted to be!
Try not to bump into corpsmen. Your life might depend on it.
With a cigarette pack in hand, target the mouth and click yourself to instantly take one.

> Spamming flares can be incredibly helpful, especially when Kings use Nightfall! Many marines cannot see in darkness, so if you lighten up the firing lane with flares, marines can shoot better.

Reviewer 2: ```suggestion
Spamming flares can be incredibly helpful, especially when the King uses Nightfall! Many marines cannot see in darkness, so if you lighten up the firing lane with flares, marines can shoot better.
```

The cryopods in medbay heal most types of damage but not organ damage!
When in doubt, throw them in the autodoc.

> Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors or replace the power cell in medbay's APC.

Reviewer 2: ```suggestion
Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors or replace the power cell in the medbay APC.
```
